deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -2836,3 +2837,16 @@ extension ChatViewController: SendContactViewControllerDelegate {
         stageVCard(url: vcardURL)
     }
 }
+
+// MARK: - ReactionsOverviewViewControllerDelegate
+
+extension ChatViewController: ReactionsOverviewViewControllerDelegate {
+    func showContact(_ viewController: UIViewController, with contactId: Int) {
+        // dismiss, but push contact-screen
+
```

Comment:
comments, that just repeat _what_ happens in the next lines, are usually superfluous, as this is obvious.

commenting the _why_ is usually better, but well, in this case it seems not needed as well, we're not doing that at similar places as well.

therefore i would just remove the comment. so it also cannot get stale on code changes.

```suggestion
```

Reply:
ah, damn, it's a leftover from development, I just didn't remove it. Busted!

Good, that you caught that.